src/SFA.DAS.ApprenticeCommitments.Jobs/IEcsApi.cs

```diff
@@ -0,0 +1,11 @@
+﻿using RestEase;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeCommitments.Jobs
+{
+    public interface IEcsApi
+    {
+        [Post("apprenticeships")]
+        Task CreateApprentice([Body] NewApprenticeship email);
```

Comment:
I do like this! But how does it know which Json convention to follow etc? (camel or pascal case)

Reply:
[By default](https://github.com/canton7/RestEase#controlling-serialization-and-deserialization) it uses Json.Net's standard JSON serialisation, we can configure it if necessary.  e.g. I did that in [another project](https://github.com/SkillsFundingAgency/das-zendesk-monitor/blob/735bfff45853e8be52be38915536ea0a3da53583/src/SFA.DAS.Zendesk.Monitor/Middleware/ApiFactory.cs#L32-L39).